nixos/modules/services/networking/syncthing.nix

```diff
@@ -429,12 +429,27 @@ in {
 
   config = mkIf cfg.enable {
 
+    environment.systemPackages = lib.optional config.xdg.menus.enable
```

Comment:
I believe makeDesktopItem runs some additionnal checks and thus should be used.

Reply:
Right done.